unified_planning/model/metrics.py

```diff
@@ -14,15 +14,28 @@
 #
 
 import unified_planning as up
+from typing import Dict
 
 
 class PlanQualityMetric():
     '''This is the base class of any metric for plan quality'''
     pass
 
 class MinimizeActionCosts(PlanQualityMetric):
+    def __init__(self, costs: Dict['up.model.Action', 'up.model.FNode']):
```

Comment:
Shall we add a getter method like `get_action_cost(a:Action) -> FNode`?

Reply:
Yes, we can add it.